apps/src/templates/levelSummary/CheckForUnderstanding.jsx

```diff
@@ -88,10 +83,8 @@ const CheckForUnderstanding = ({
           <p>
             <i className="fa fa-user" />
             <span>
-              {i18n.studentsSubmitted({
```

Comment:
Is this string no longer needed? Can it be removed?

Reply:
yeah, i wasn't sure if there were rules around when we remove things from the strings. it's unused though so i can take it out if that doesn't break anything

Reply:
If it's definitely unused, it's okay to remove. For JS strings, the page will crash if the string isn't found, so we just want to be sure it's definitely not being used